docs/monitors/windows_event_log_monitor.md

```diff
@@ -86,3 +86,4 @@ and System sources:
 | `remote_user`                | Optional (defaults to ``None``). The username to use for authentication on the remote server.  This option is only valid on Windows Vista and above. | 
 | `remote_password`            | Optional (defaults to ``None``). The password to use for authentication on the remote server.  This option is only valid on Windows Vista and above. | 
 | `remote_domain`              | Optional (defaults to ``None``). The domain to which the remote user account belongs.  This option is only valid on Windows Vista and above. | 
+| `json`                       | Optional (defaults to ``False``). Format events as json? Supports inclusion of all event fields. This option is only valid on Windows Vista and above. | 
```

Comment:
We should also figure out / decide if ``true`` should be the default value.

Reply:
From a conservation early on with Dario, the intention is this would be false initially to be backwards-compatible for  queries/dashboards customers have built but ultimately it would become the default (or the NewJsonApi impl. would replace the NewApi impl.)